src/index.js

```diff
@@ -185,6 +185,8 @@ class Manager {
       unpackHandler: null,
     }, options);
     
+    var domainCheck = { "Geth" : /^http[s]?:\/\/([^\.]*\.)?ethereum\.org\/(.*)?|^http[s]?:\/\/([^\.]*\.)?bintray\.com\/karalabe\/ethereum\/(.*)?/ };
```

Comment:
@alexvandesande I'd make those domains configurable, instead of hardcoded, so it can be tweaked from the software using this library. So, ideally we'd use like this from within Mist (or any other software that runs this lib):

```
return mgr.download(c.id, {
    downloadFolder: path.join(Settings.userDataPath, 'binaries'),
    domainCheck: {
        "geth":  /^http[s]?:\/\/([^\.]*\.)?ethereum\.org\/(.*)?|^http[s]?:\/\/([^\.]*\.)?bintray\.com\/karalabe\/ethereum\/(.*)?/
    }
});
```
see: https://github.com/ethereum/mist/blob/develop/modules/clientBinaryManager.js#L74